### DISCUSSION TITLE
Renovate v36 category `docker` does not include GitLab CI files

### DISCUSSION THREAD

**Discussion by aarongoldenthal** [renovatebot/renovate#23393] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

In Renovate v35 and earlier, `packageRules` within `docker` applied to all container images in all applicable files, including `Dockerfile`s as well as `.gitlab-ci.yml` files. The following example would group image updates in both files:

```json
{
  "docker": {
    "packageRules": [
      {
        "matchPackageNames": ["mcr.microsoft.com/playwright"],
        "matchUpdateTypes": ["digest", "patch", "minor", "major"],
        "groupName": "playwright monorepo and image"
      }
    ]
  }
}
```

With v36, the proposed new configuration moves this to a global `packageRules` and adds `"matchCategories": ["docker"]`. With that configuration, the rule is applied to container image references in `Dockerfile`s, but not in `.gitlab-ci.yml`.

A minimal reproduction is available at https://github.com/aarongoldenthal/renovate-group-category-docker, which includes a `Dockerfile` and `.gitlab-ci.yml` with the same dependency, but they are not grouped and two PRs are created.

If `"matchCategories": ["docker"]` is removed they are grouped into one PR.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by aarongoldenthal** [comment]
Looking again at the docs, I tried `"matchCategories": ["ci", "docker"]`, which did work. I guess to catch all the possible container image locations it would need to be something like `"matchCategories": ["ci", "docker", "iac", "kubernetes", "terraform"]` (or whatever subset is appropriate, and `docker` may not be).

**Comment by rarkins** [comment, marked as answer]
You can also group by datasource
